schema_utils_test.py

```diff
@@ -53,13 +53,16 @@
 SCHEMA_TYPE_INT = schema_utils.SCHEMA_TYPE_INT
 SCHEMA_TYPE_LIST = schema_utils.SCHEMA_TYPE_LIST
 SCHEMA_TYPE_UNICODE = schema_utils.SCHEMA_TYPE_UNICODE
+SCHEMA_TYPE_OPTIONAL_UNICODE = schema_utils.SCHEMA_TYPE_OPTIONAL_UNICODE
```

Comment:
There should be lots of tests for normalization of "optional unicode". Where are they?